### PR TITLE
fix[UI]: change the color and contents for reset data

### LINKED ISSUE #5825: [UI] Database Reset: Use dark red button color
#### Current Behavior
The database reset action carries the high severity of data loss and should be visually represented in this way to the user.

Screen Shot 2022-07-14 at 6 34 02 PM

#### Implementation
1. Use the dark red theme color for the "PROCEED" button. See design language in Figma (or the color of the Undeploy button).
1. Use "RESET" as the button text, not "PROCEED".

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by techwithbhavya**
Signed-off-by: bhavyastar <bhavya.sachdeva.58@gmail.com>

**Description**
-- As given in the issue, I replaced PROCEED with RESET
-- Change the background color of PROCEED which is now RESET to dark red color (#ff3333)

Changes : 
image

This PR fixes #5825 

**Notes for Reviewers**
 Thanks in advance for reviewing my PR. Happy to contribute to this and looking forward to contributing more.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
 Yes, I signed my commits.

### CODE REVIEW

File: ui/components/PromptComponent.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ const styles = (theme) => ({
     margin : theme.spacing(0.5),
     padding : theme.spacing(1),
     borderRadius : 5,
+    backgroundColor : "#ff3333",
```
Comment (leecalcote) [inline ui/components/PromptComponent.js:35]:
```suggestion
    backgroundColor : "#B32700",
```

File: ui/components/PromptComponent.js [hunk 2]
```diff
@@ -32,6 +32,11 @@ const styles = (theme) => ({
...
+    "&:hover" : {
+      backgroundColor : "#ff3333",
```
Comment (leecalcote) [inline ui/components/PromptComponent.js:37]:
```suggestion
      backgroundColor : "#B32700",
```

**Review by leecalcote — COMMENTED** [review]
Good stuff, @bhavyastar 👍 

FYI - project colors can be found under the "Design Language" section of "Meshery UI" Figma project.

Screen Shot 2022-07-19 at 5 01 21 PM

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Yay, @bhavyastar! 🎈

**Comment by bhavyastar** [comment]
Thanks, Lee, Actually earlier I used this color but then used the color of the Undeploy button. From now, I will definitely confirm with you in order to proceed with PR. 
Yay!! 🎈🎉 Thanks a lot @leecalcote.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Comment by leecalcote** [comment]
You’re right - the colors are a bit mixed up in different places. I looked again after merging and I think you made the right call. 

We really need to be consistently referencing a theme. There is an active effort to overhaul Meshery UI that might be of interest to you. See @YashKamboj or MeshMate @Chadha93 for details, if so.

**Pull request merged**